commands/proposal_submission.go

```diff
@@ -975,6 +977,29 @@ func checkPriceMonitoring(parameters *protoTypes.PriceMonitoringParameters, pare
 	return errs
 }
 
+func checkLiquidationStrategy(params *protoTypes.LiquidationStrategy, parent string) Errors {
+	errs := NewErrors()
+	if params == nil {
+		// @TODO these will be required, in that case the check for nil should be removed
+		// or return an error.
+		return errs
+	}
+	dispFrac, err := num.DecimalFromString(params.DisposalFraction)
+	if err != nil || dispFrac.IsNegative() || dispFrac.IsZero() || dispFrac.GreaterThan(num.DecimalZero()) {
+		errs.AddForProperty(fmt.Sprintf("%s.disposal_fraction", parent), ErrMustBeBetween01)
+	}
+	maxFrac, err := num.DecimalFromString(params.MaxFractionConsumed)
+	if err != nil || maxFrac.IsNegative() || maxFrac.IsZero() || maxFrac.GreaterThan(num.DecimalZero()) {
+		errs.AddForProperty(fmt.Sprintf("%s.max_fraction_consumed", parent), ErrMustBeBetween01)
+	}
+	if params.DisposalTimeStep < 1 {
+		errs.AddForProperty(fmt.Sprintf("%s.disposal_time_step", parent), ErrMustBePositiveOrZero)
```

Comment:
Is `ErrMustBePositiveOrZero` correct here?

Reply:
Not anymore, fixed